hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirAttrOp.java

```diff
@@ -342,7 +342,7 @@ static INodeDirectory unprotectedSetQuota(
     if ((type != null) && (!fsd.isQuotaByStorageTypeEnabled() ||
         nsQuota != HdfsConstants.QUOTA_DONT_SET)) {
       throw new UnsupportedActionException(
-          "Failed to set quota by storage type because either" +
+          "Failed to set quota by storage type because either " +
```

Comment:
Hi @hfutatzhanghb  IMO, It is best not to modify it here. It is not very relevant to this function and will increase the pipeline time.

Reply:
@KeeProMise Thanks a lot. Have fixed.